packages/client/tests/functional/extensions/query.ts

```diff
@@ -529,121 +529,124 @@ testMatrix.setupTestSuite(
       },
     )
 
-    testIf(provider !== Providers.MONGODB && process.platform !== 'win32')(
-      'hijacking a batch transaction into another one with a simple call',
-      async () => {
-        const fnEmitter = jest.fn()
+    testIf(
+      provider !== Providers.MONGODB &&
+        process.platform !== 'win32' &&
+        providerFlavor !== ProviderFlavors.JS_PLANETSCALE,
```

Comment:
Add a comment here for why the test is skipped with `providerFlavor !== ProviderFlavors.JS_PLANETSCALE`?